src/yival/cli/run.py

```diff
@@ -36,7 +36,7 @@ def add_arguments_to(subparser):
     parser.add_argument(
         "--interactive",
         action="store_true",
-        default=False,
+        default=True,
```

Comment:
Why set interactive=True here 🤔

Reply:
Good catch, I forgot to change back when I was testing. Reseted